From d954e80418d2bd10e657c665031b3c38b30fbd35 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 13 Mar 2017 23:35:56 +0100
Subject: [PATCH] mesonlib: Use 'lib' as default libdir

Everything except for the linker installs to /usr/lib, so that's
what we want to use for aesthetic reasons as well.
---
 mesonbuild/mesonlib.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mesonbuild/mesonlib.py b/mesonbuild/mesonlib.py
index 551bda2e..3917d03f 100644
--- a/mesonbuild/mesonlib.py
+++ b/mesonbuild/mesonlib.py
@@ -222,8 +222,6 @@ def default_libdir():
                 return 'lib/' + archpath
         except Exception:
             pass
-    if os.path.isdir('/usr/lib64') and not os.path.islink('/usr/lib64'):
-        return 'lib64'
     return 'lib'
 
 def default_libexecdir():
-- 
2.12.0

